external-docs/docs/guides/shape-data.md

```diff
@@ -1,3 +1,241 @@
 # Shaping your data
 
-todo: Explain the shapes of data we accept. Issue #17.
+> This page discusses the shapes of input data supported by csvcubed with the aim to help you understand which to use and how to generate data in the right shape.  
+
+csvcubed requires that all CSV data inputs are provided in one of two specialised form of [tidy data](../glossary/index.md#tidy-data):
+
+* the [canonical approach](#canonical-shape) - the **default recommended shape** accepted by csvcubed. It is the **most flexible** but also the most **verbose**.
+* the [pivoted approach](#pivoted-shape) - a **terser** shape currently only compatible with data sets containing a **single measure**.
+
+These two shapes share a number of similarities in how they require data to be structured; this is explored in the following section on the [common structure](#common-structure).
+
+## Common Structure
```

Comment:
Can we provide a json example of this dataset - like done in the Canonical Shape section?

```json
{
    "$schema": "http://purl.org/csv-cubed/qube-config/v1.0",
    "title": "'Arthur's Bakes' stores in UK cities from 2020 to 2022",
    "description": "The number of 'Arthurs' Bakes' stores in cities across the UK between 2020 and 2022.",
    "creator": "HM Revenue & Customs",
    "publisher": "HM Revenue & Customs",
    "columns": {
        "Year": {
            "type": "dimension"
        },
        "Location": {
            "type": "dimension"
        },
        "Value": {
            "type": "observations"
        },
        "Status": {
            "type": "attribute"
        }
    }
}